src/test_validator.py

```diff
@@ -0,0 +1,180 @@
+import unittest
+import datetime as dt
+from validation import String, Date
+
+def broken_function():
+    raise Exception('This is broken')
+
+class TestValidator(unittest.TestCase):
+    
```

Comment:
Very minor thing but at the beginning of functions I believe you don't need an empty line between the def and the first method.